### ISSUE TITLE
Provide Permission to enable/disable the Menu entry

### ISSUE THREAD

**Issue by marcelfischer** [Icinga/icingaweb2-module-graphite#250]
When I use Menu -> Graphite -> Services it generates huge load on graphite-web, icingaweb2 and my web browser because it loads many graphs depending on my "Default page size". For example 100 entries...
I don't really see the use-case of loading random 25 or 100 graphs.

## Expected Behavior
I would be nice to have an permission point in the graphite module to enable/disable the menu entry.
Maybe graphs should also only load when someone uses the search bar and there should be a hard limit of api calls somehow.

## Current Behavior
A "Default page size" of 100 leads to a not responding icinga web and generates high load on the graphite-web server.

## Your Environment

* Module version (System - About): latest git commit 
* Icinga Web 2 version and modules (System - About): 2.8.1
* Graphite (carbon, web, whisper) version: graphite-web 1.1.7

**Comment by carraroj** [comment]
ref/NC/701309

**Comment by lippserd** [comment]
I don't think that a permission makes sense here. It's actually about exactly that:

> I don't really see the use-case of loading random N graphs.

We introduced a limit of two graphs a while ago, but that doesn't seem to work anymore. I think we should just fix this.

**Comment by Tqnsls** [comment]
Hi,
the `graphs_limit=2`, which is set as default in the module, works, but it only limits the maximum graphs per host / service not the "Default page size" :)
You could change the default values in the configuration.php of the module and for example append something like `&host=` to prohibit loading any graphs initially.
But customizing the file could be overwritten when updating the module, of course.

**Comment by sukhwinder33445** [comment]
Only the required graphs are fetched. The default value `graphs_limit` per object is set to 2, so only the first `25x2` graphs (or less) are fetched.
